docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
     restart: always
     logging: *default-logging
   dispatcher:
-    image: semtech/mu-dispatcher:1.1.2
+    image: semtech/mu-dispatcher:2.1.0-beta.1
```

Comment:
The reason I went with the latest is because of the reverse host match functionality and the new logging layer but this is not really something we NEED to have at the moment. I can change it to v2.0.0 until the stable version of v2.1.0 comes out :)

Reply:
update: define_layers is only available in v2.1.0-beta.1. Using v2.0.0 breaks everything